ingredients/LSSTCam-imSim/DRP.yaml

```diff
@@ -207,6 +207,20 @@ subsets:
       This step has global aggregation tasks to run over all visits, detectors,
       tracts, etc.  This step should be run only with the instrument constraint
       in the data query.
+  step8:
+    subset:
+      - analyzeObjectTableCore
+      - analyzeObjectTableSurveyCore
+      - catalogMatchTract
+      - refCatObjectTract
+      - diff_matched_analysis
+    description: |
+      Core (most important), coadd-level plots/metrics from analysis_tools.
```

Comment:
All the other steps have instructions on what they can or cannot put in the data query. Make it clear that as long as "analyzeObjectTableSurveyCore" is here, that this is a global task and they may not put anything in the data query or split up into tracts